index.js

```diff
@@ -29,20 +29,15 @@ function getJSON(obj) {
 		} else {
 			obj.headers = {'User-Agent': FAKE_USER_AGENT};
 		}
-		request({
-			url: obj.url,
-			headers: obj.headers,
-			rejectUnauthorized: false
-		}, function (err, req, body) {
-			if (err) {
-				return reject(err);
-			}
+		got(obj.url, {
+			headers: obj.headers
+		}).then(request => {
```

Comment:
That was a simple mess up on my end, I made all the changes and then accidentally discarded them all and just typed it back up real quick, must've slipped by me.